.github/workflows/check-contribution.yml

```diff
@@ -0,0 +1,16 @@
+name: Check PR preview
+
+on:
+  pull_request:
+jobs:
+  check_contribution:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write # surge-preview write PR comments
```

Comment:
**nitpick:** not for surge, but we need it for the `pr-antora-content-guidelines-checker` action 😸